gnome/gstreamer1-gst-plugins-bad/Portfile

```diff
@@ -142,21 +140,41 @@ variant fdkaac description {Enable fdkaac AAC plugin} {
 }
 
 # Window system is provided by gstreamer1-gst-plugins-base, so sync up.
-variant x11 {
+variant x11 conflicts applemedia {
     require_active_variants port:gstreamer1-gst-plugins-base x11
 }
 
-default_variants +x11
```

Comment:
You seem to have removed the default variant entirely?

Reply:
It is just moved to line 163.  The default is still +x11 as before if the -base port is either not active yet or its +x11 variant is active (which is its default).  The default is +applemedia if this port and the -base port are both -x11 -universal on 10.9 and later (i.e. if it will work).  If the -base port is -x11 but the requirements aren't met for applemedia then neither x11 nor applemedia will work and the default is no variants.